views/macros/renderPartial.njk

```diff
@@ -1,17 +1,19 @@
 {% set _loadedStylesheets = [] %}
 
 {% macro renderPartial(partial) %}
+    {% if env.dev %}
```

Comment:
This actually has the side effect of being hidden on staging too, which can sometimes be helpful.

Two thoughts:

1. Should we create a new property of the environment called `prod` which can be set? This would be a little more like webpack, but also different because an environment could technically not be `dev` nor be `prod`.
2. We could just align around a naming convention, and have an environment name be `prod`, and then the check would be:

```
{% if env.name != 'prod' %}
...
{% endif %}
```

Reply:
Another thought, is that we are planning on having params be configured in the editor (ex: `?help=true`) that the editor would be able to pass, so we could use the request params as well to determine when to show the help text.